### PR TITLE
Add link to changelog on PyPI page

### PR DESCRIPTION
**Pull request by A5rocks**
I'll make a test release on TestPyPI when I feel like it (probably in a few days :( ). I think this should work, though.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -118,4 +120,8 @@
         "Topic :: System :: Networking",
         "Framework :: Trio",
     ],
+    project_urls={
+        "Documentation": "https://trio.readthedocs.io/"
```
Comment (A5rocks) [inline setup.py:124]:
```suggestion
        "Documentation": "https://trio.readthedocs.io/",
```

**Review by A5rocks — COMMENTED** [review]
Changing code is weird on mobile

File: setup.py [hunk 2]
```diff
@@ -118,4 +120,8 @@
         "Topic :: System :: Networking",
         "Framework :: Trio",
     ],
+    project_urls={
+        "Documentation": "https://trio.readthedocs.io/",
+        "Changelog": "https://trio.readthedocs.io/en/stable/history.html"
```
Comment (A5rocks) [inline setup.py:125]:
```suggestion
        "Changelog": "https://trio.readthedocs.io/en/stable/history.html",
```

**Review by A5rocks — COMMENTED** [review]
:(

**Review by CoolCat467 — APPROVED** [review]
Looks good to me

**Review by TeamSpen210 — APPROVED** [review]

**Review by jakkdl — APPROVED** [review]
Because you clearly need more approvals before you dare releasing this :p

**Comment by A5rocks** [comment]
Tested (before doing `stable` -> `latest`), looks fine: https://test.pypi.org/project/A5rocks-trio/0.22.3/

**Pull request merged**